src/ffi.md

```diff
@@ -21,6 +21,29 @@ libc = "0.2.0"
 
 [libc]: https://crates.io/crates/libc
 
+## Prepare the build script
+
+Because [snappy](https://github.com/google/snappy) is a static library by default. So there is no C++ std linked in the output artifact. In order to use this foreign library in Rust, we have to manually specify that we want to link stdc++ in our project. The easiest way to do this is by setting up a build script.
```

Comment:
```suggestion
Because [snappy](https://github.com/google/snappy) is a static library by default.
So there is no C++ std linked in the output artifact. 
n order to use this foreign library in Rust, we have to manually specify that we want to link stdc++ in our project.
The easiest way to do this is by setting up a build script.
```